tests/test_wcs.cc

```diff
@@ -12,26 +12,26 @@
 #include "lsst/afw/image/TanWcs.h"
 #include "lsst/afw/image/Utils.h"
 #include "lsst/afw/fits.h"
-#include "lsst/daf/base.h" 
+#include "lsst/daf/base.h"
 
 #include <stdlib.h> /* for getenv */
 
+// NOTE: turn this flag on to raise exceptions on floating point errors.
+// NOTE: this only works on GNU/Linux (fenv is not C++ standard).
```

Comment:
`fenv.h` is a standard include file from the C99 floating point environment. What's non-standard is the Gnu `feenableexcept` function.